tests/test_wsireader.py

```diff
@@ -531,6 +577,30 @@ def test_read_rect_jp2_baseline(sample_jp2):
     assert im_region.shape == (*size[::-1], 3)
 
 
+def test_read_rect_tiffreader_svs_baseline(sample_svs):
+    """Test TIFFWSIReader.read_rect with an SVS file at baseline."""
+    wsi = wsireader.TIFFWSIReader(sample_svs)
+    location = SVS_TEST_TISSUE_LOCATION
+    size = SVS_TEST_TISSUE_SIZE
+    im_region = wsi.read_rect(location, size, resolution=0, units="level")
+
+    assert isinstance(im_region, np.ndarray)
+    assert im_region.dtype == "uint8"
+    assert im_region.shape == (*size[::-1], 3)
+
+
+def test_read_rect_tiffreader_ome_tiff_baseline(sample_ome_tiff):
+    """Test TIFFWSIReader.read_rect with an OME-TIFF file at baseline."""
+    wsi = wsireader.TIFFWSIReader(sample_ome_tiff)
```

Comment:
you can use get_wsireader here.

Reply:
To be honest, I am trying to avoid it as it is not pythonic. I really don't like using a function to construct an object like this. I think you should be calling a class.